rasa/importers/autoconfig.py

```diff
@@ -0,0 +1,240 @@
+import copy
+import logging
+import os
+
+from typing import Text, Dict, Any, List
+
+from rasa.cli import utils as cli_utils
+from rasa.constants import (
+    CONFIG_AUTOCONFIGURABLE_KEYS,
+    DOCS_URL_PIPELINE,
+    DOCS_URL_POLICIES,
+    CONFIG_KEYS,
+)
+from rasa.utils import io as io_utils, common as common_utils
+
+logger = logging.getLogger(__name__)
+
+COMMENTS_FOR_KEYS = {
+    "pipeline": (
+        f"# # No configuration for the NLU pipeline was provided. The following "
+        f"default pipeline was used to train your model.\n"
+        f"# # To customise it, uncomment and adjust the pipeline. See "
```

Comment:
Do we also want to let the user know explicitly that they can leave it as is if they want? For some reason this makes me feel like i have to uncomment and adjust it, but maybe that's just me 😄 Maybe instead of `To customize it` we could do `If you'd like to customize it` (or want, need)